to_xls.py

```diff
@@ -28,14 +28,14 @@
 # Helper method to clean up cell values by
 #  turning 'undefined' and 'null' into empty cell
 def clean_undefined(x) -> str:
-    if "undefined" in x:
-        return ''
-    if "null" in x:
-        return ''
+    if isinstance(x, (str, list, tuple, set, dict)):
```

Comment:
Did you see the original code crashing? 
if so, we should perhaps also handle the `not ( isinstance ... ` case (?)

Reply:
I think upon further inspection I can fix the error i was seeing more simply. I'll try it out